### DISCUSSION TITLE
Excellent Product - Any Limits to Database Size?

### DISCUSSION THREAD

**Discussion by RayBright** [massCodeIO/massCode#379] [General]
To date I've been using a program called 'Snippets (Version 0.8.2)' by a Uladzimir Prudnikau  to store my bits of Arduino code but this program no longer has a home page or any development since about 2012 and has no capability to download or backup my snippets.

MassCode apears to overcome these problems for me BUT can someone advise if there is a limit to the number of saved folders and snippets OR as seeming to be a json database is this unlimitted.  Comments appreciated

**Comment by antonreshetov** [comment]
massCode uses json-server to read and write data in JSON format. Theoretically, there are no limits on the number of saved folders and code fragments, but performance may suffer with a large amount of data.

https://github.com/typicode/json-server/issues/529
